### ISSUE TITLE
Failed images don't offer image-related context menu actions

### ISSUE THREAD

**Issue by rulatir** [brave/brave-browser#25876]
## Description 

Broken/failed images (displaying the broken image placeholder and alt text) no longer offer image-related actions in the context menu.

## Steps to Reproduce

   1. Go to this codepen: https://codepen.io/makaroni4/pen/WLzgpN
   2. Right click on the image placeholder in the output frame

## Actual result:

A context menu without image-related actions.

## Expected result:

A context menu with image-related actions.

## Reproduces how often: 

Always.

## Brave version (brave://version info)

Brave	1.44.108 Chromium: 106.0.5249.103 (Oficjalna wersja) unknown (64-bitowa) 
Revision	[REDACTED]-refs/branch-heads/5249_91@{#6}
OS Linux

(Regarding the redaction: sorry, I'm not sending this amount of opaque data; searching for that hex string as a commit hash in this repository yields no results, so I don't know what information is encoded in this field).

## Version/Channel Information:

I don't know what "channel" means in this context.

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? No

## Miscellaneous Information:

Rationale for fixing: something as simple as "copy image location" to report a 404 image to website maintainer is made much harder by this bug (requires using devtools).

**Comment by Sayantan19** [comment]
I could not understand what exactly is supposed to be shown. A menu does appear when I right click on the broken image icon, with options like load image and all. Can you please elaborate on it? I have attached a screenshot of the same too!
![Screenshot (4)](https://user-images.githubusercontent.com/75779600/194758693-4f3faafe-a22e-414f-abf3-63f5a7c07679.png)
